test/forwarddiff.jl

```diff
@@ -115,3 +115,16 @@ end
     @show derivative_ε derivative_fd
     @test norm(derivative_ε - derivative_fd) < 1e-4
 end
+
+@testset "Derivative of complex function" begin
+    using SpecialFunctions, FiniteDifferences
+
+    tol = 1e-10
+    fact=randn(ComplexF64)
+    value=randn()
```

Comment:
spacing

Reply:
also you could probably use more helpful variable names (you may use more than 5 characters :smile:)

Reply:
Well you also have a dead line, which I just removed.

Reply:
Certainly cleaner now, though!